index.js

```diff
@@ -0,0 +1,113 @@
+            <div class="itemsA">
+              <h5>SohYeong Noh</h5>
+              <span
+                >Director of Art Centre Nabi and a board member of CC
+                Korea</span
+              >
+              <p>
+                As the main venue for new media art production in Korea, Nabi
+                promotes cross-disciplinary collaboration and understanding
+                among science technology, humanities, and the arts.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="speakers2">
+          <div class="speakeritems">
+            <img src="./images/speakers/speaker_03.png" alt="" />
+            <div class="itemsA">
+              <h5>Lila tretikov</h5>
+              <span>Executive Director of the Wikimedia Foundation</span>
+              <p>
+                Lila Tretikov is the Executive of the Wikimedia Foundation, the
+                nonprofit organization that operates Wikipedia. Wikipedia is
+                freely available in 290 languag-es and used by nearly half a
+                billion people around the world every month.
+              </p>
+            </div>
+          </div>
+          <div class="speakeritems">
+            <img src="./images/speakers/speaker_04.png" alt="" />
+            <div class="itemsA">
+              <h5>Kilnam Chon</h5>
+              <span></span>
+              <p>
+                Kilnam Chon helped bring the internet to Asia and is an
+                outspoken advocate for the open web and digital com-mons. In
+                2012. he was inducted into the inaugural class of the Internet
+                Society’s (ISOC) Internet Hall of Fame
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="speakers3">
+          <div class="speakeritems">
+            <img src="./images/speakers/speaker_05.png" alt="" />
+            <div class="itemsA">
+              <h5>Julia Leda</h5>
+              <span>President of Young Pirates of Europe</span>
+              <p>
+                European ingetration, political democracy and participation of
+                youth through online as her major condern, Reda’s report
+                outlining potential changes to EU copyright law was approved by
+                the Parliament in July
+              </p>
+            </div>
+          </div>
+          <div class="speakeritems">
+            <img src="./images/speakers/speaker_06.png" alt="" />
+            <div class="itemsA">
+              <h5>Ryan Merkley</h5>
+              <span
+                >CEO of Creativve Commons, ex COO of the Mozilla
+                Foundation</span
+              >
+              <p>
+                Ryan had been leading open-source projects at the Mozilla
+                Foundation such as the open-source move-ment
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+`;
```

Comment:
- [ ] Kindly adhere to the project requirement, as per the requirement you should need to render the speakers dynamically by creating an array object of data of speakers and then iterate over that array to render the things dynamically. But as I can see you dynamically rendered the speaker section but the way you implemented the things is not correct. Please follow the requirements as I have attached the screenshot and make the changes.

### Requirement
![image](https://github.com/enocol/capstone/assets/66476329/f23fa5f7-bfe7-4229-ba1d-a24b1f619dc2)

Reply:
@skyv26 Thanks for your comments. I also wish to refer to the requirement which states that, You should use a JavaScript **variable** with the data about the speakers and use it when the page loads to create the HTML for this section dynamically. In my project,  I  used a javascript variable called 'content' to store the data which is dynamically rendered when the pages load. In software development, there is no single method to solve a problem or a method cannot be referred to as **not correct.**. It's very unprofessional to say a method is not correct to solve a problem.

Reply:
@enocol Please take the second opinion if you think that review is not fair. I can understand that you put a lot of effort. I didn't mean that you are totally wrong but it might be possible that you misunderstood the requirements.

Reply:
Your review has totally put me off. I have seen projects far worst but approved so far. I don't even know where to start to implement some of your recommendations. The project requirement mentioned that it should closely **resemble** not exactly,ly and use a different topic or content. How can you be comparing two things in the manner you did with different content? My project was highly customized as recommended, how do I start to change things now?

Reply:
@enocol Please ask for a second opinion by clicking [here](https://airtable.com/shrQAqnBwek5a0O0s), because as a code reviewer, it is my duty to make sure that you are fulfilling the requirements or not. Calling me **unprofessional** is not going to help you to pass the block.